tests/features/support/scope.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
       scope.page.waitForSelector('.alert-danger'),  // Not sure this is useful with url detection now
       scope.page.waitForSelector('.da-has-error'),
       Promise.all([
-        scope.page[ scope.device_map[ scope.emulating ]]('button.btn-primary[type="submit"]'),
+        // scope.page[ scope.device_map[ scope.emulating ]](button_selector),
```

Comment:
Do we still need this comment in here? I found it confusing at first glance because it seemed like we removed some old functionality here.

Reply:
Completely removable. It's actually part of a possible future.